scripts/arm32_ci_script.sh

```diff
@@ -200,28 +200,32 @@ function cross_build_core_setup_with_docker {
     else
         exit_with_error "ERROR: unknown buildArch $__buildArch" false
     fi
-
-    # Build rootfs
     __dockerCmd="sudo docker run --privileged -i --rm -v $__currentWorkingDirectory:/opt/core-setup -w /opt/core-setup $__dockerImage"
-    __buildRootfsCmd="./cross/build-rootfs.sh $__buildArch $__linuxCodeName --skipunmount"
 
-    (set +x; echo "Build RootFS for $__buildArch $__linuxCodeName")
-    $__dockerCmd $__buildRootfsCmd
-    sudo chown -R $(id -u -n) cross/rootfs
+    if [ $__skipRootFS == 0 ]; then
```

Comment:
In case of armel, it still generates rootfs in docker image. So I fixed some code. Thank you.